openapi.yaml

```
@@ -81,12 +81,13 @@ paths:
           $ref: '#/components/responses/ServerError'
 
     put:
-      summary: ""
+      summary: Обновить информацию о пользователях
```

Метод работает не так. Он изменяет только одного пользователя.
```suggestion
      summary: Обновить информацию о пользователе по ID
```